test/e2e/serial/tests/configuration.go

```diff
@@ -1481,6 +1481,33 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				}
 			})
 		})
+
+		It("should set terminationMessagePolicy in the operand", Label(label.Tier2, "feature:introspection"), func(ctx context.Context) {
+			By("getting NROP object")
+			nroKey := objects.NROObjectKey()
+			nroOperObj := nropv1.NUMAResourcesOperator{}
+
+			err := fxt.Client.Get(ctx, nroKey, &nroOperObj)
+			Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", nroKey.String())
+
+			if len(nroOperObj.Spec.NodeGroups) != 1 {
+				// TODO: this is the simplest case, there is no hard requirement really
+				// but we took the simplest option atm
+				e2efixture.Skipf(fxt, "more than one NodeGroup not yet supported, found %d", len(nroOperObj.Spec.NodeGroups))
+			}
```

Comment:
I'm aware that this is referring to support in tests not to the operator support, but I don't think we need this block  anyway. There is a 1:1 relation between nodegroup: MCP and thus a 1:1 relation between nodeGroup:RTE-DS, so the loop below should cover single/multiple nodegroup cases.